public/app/plugins/datasource/loki/LanguageProvider.ts

```diff
@@ -1,77 +1,25 @@
-import { chain, difference } from 'lodash';
 import { LRUCache } from 'lru-cache';
-import Prism, { Grammar } from 'prismjs';
+import Prism from 'prismjs';
 
-import { dateTime, AbsoluteTimeRange, LanguageProvider, HistoryItem, AbstractQuery, KeyValue } from '@grafana/data';
-import { CompletionItem, TypeaheadInput, TypeaheadOutput, CompletionItemGroup } from '@grafana/ui';
-import {
-  extractLabelMatchers,
-  parseSelector,
-  processLabels,
-  toPromLikeExpr,
-} from 'app/plugins/datasource/prometheus/language_utils';
+import { LanguageProvider, AbstractQuery, KeyValue } from '@grafana/data';
+import { extractLabelMatchers, processLabels, toPromLikeExpr } from 'app/plugins/datasource/prometheus/language_utils';
 
 import { LokiDatasource } from './datasource';
 import {
   extractLabelKeysFromDataFrame,
   extractLogParserFromDataFrame,
   extractUnwrapLabelKeysFromDataFrame,
 } from './responseUtils';
-import syntax, { FUNCTIONS, PIPE_PARSERS, PIPE_OPERATORS } from './syntax';
+import syntax from './syntax';
 import { ParserAndLabelKeysResult, LokiQuery, LokiQueryType } from './types';
 
-const DEFAULT_KEYS = ['job', 'namespace'];
-const EMPTY_SELECTOR = '{}';
-const HISTORY_ITEM_COUNT = 10;
-const HISTORY_COUNT_CUTOFF = 1000 * 60 * 60 * 24; // 24h
 const NS_IN_MS = 1000000;
 
-// When changing RATE_RANGES, check if Prometheus/PromQL ranges should be changed too
-// @see public/app/plugins/datasource/prometheus/promql.ts
-const RATE_RANGES: CompletionItem[] = [
```

Comment:
CompletionItem is used in slate editor that we don't use anymore, as we replaced it with Monaco. 
<img width="1214" alt="image" src="https://github.com/grafana/grafana/assets/30407135/5782cf21-ab7a-4f89-adbd-74b102fbd4ba">

I have double checked if autocompletion + completion in query builder still works as expected. 

<img width="436" alt="image" src="https://github.com/grafana/grafana/assets/30407135/c1784bca-c085-4637-93f9-68c1400593c8">

<img width="923" alt="image" src="https://github.com/grafana/grafana/assets/30407135/f3768d25-bd0c-4b89-a719-b0672c783dec">